full/src/main/java/apoc/uuid/Uuid.java

```diff
@@ -12,8 +12,13 @@
 import java.util.Map;
 import java.util.stream.Stream;
 
+import static apoc.util.SystemDbUtil.checkWriteAllowed;
+
 @Extended
 public class Uuid {
+    private static final String MSG_DEPRECATION = "Please note that the current procedure is deprecated, \n" +
+            "it's recommended to use the `apoc.uuid.create`, `apoc.uuid.drop`, `apoc.uuid.dropAll` procedures \n" +
+            "instead of, respectively, `apoc.uuid.install`, `apoc.uuid.remove`, `apoc.uuid.removeAll`.";
```

Comment:
add "executed against the 'system' database

Reply:
Changed